### DISCUSSION TITLE
9 slice pivot y top seems to not work

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#604] [! Welcome - readme first!]
9 slice pivot y top seams to not work

**Comment by pandrr** [comment]
seams to work for me.

what does "seams to not work" mean ?

**Comment by marioepsley** [comment]
Here's an example comp. You'll see that central and bottom pivots line up, the top pivot doesnt.

https://dev.cables.gl/edit/x676iL

**Reply by pandrr** [reply]
thanks, great example!

it is now fixed on dev.cables.gl !

**Reply by marioepsley** [reply]
Just realised yesterday how useful this op can be. Great stuff

**Reply by pandrr** [reply]
yes, its a great concept. in game development this is used a lot....

**Reply by marioepsley** [reply]
If I was being very picky I would also say that the pivot y order should be the same as it is in the rectangle.
rectangle goes top/center/bottom
rectangle9Slice goes bottom/center/top
